packages/oui-skeleton/_mixins.less

```diff
@@ -0,0 +1,14 @@
+#oui {
+  .skeleton-sizes() {
```

Comment:
I think it's useless.

Reply:
We could have both. See my comment: https://github.com/ovh-ux/ovh-ui-angular/pull/185 :)